ir/inst_other.go

```diff
@@ -674,3 +674,48 @@ func (inst *InstCleanupPad) LLString() string {
 	}
 	return buf.String()
 }
+
+// ~~~ [ freeze ] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
// ~~~ [ freeze ] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Stylistic detail, for consistency these are kept at 80-character width.

Reply:
In the same way that you added `freeze` in-between `select` and `call` for `llvm/asm`, we can keep the same order in `llvm/ir`. It doesn't matter for code quality of course, but to keep a consistent order will hopefully make it easier to find things in the future.